README.md

```diff
@@ -50,6 +50,16 @@ func main {
 }
 ```
 
+### Testing
+
+This package uses [lestrrat-go/libxml2](https://github.com/lestrrat-go/libxml2) for testing purporses, which in turn depends on the libxml-2.0 C library. Ensure you have the development dependency installed. In linux this implies:
+
+```bash
+sudo apt-get install libxml2-dev
+```
+
+Tests can take a while to run as they download the complete XML documents to test against, please be patient.
+
```

Comment:
Nice addition! I'm not super-happy with how these tests work, btw. They're sometimes too slow for no clear reason, and in some cases where the XML doesn't validate the schema the library crashes rather than just returning the validation errors.